src/main/kotlin/no/nav/syfo/clients/Tilgangskontroll.kt

```diff
@@ -0,0 +1,57 @@
+package no.nav.syfo.clients
+
+import io.ktor.client.*
+import io.ktor.client.engine.cio.*
+import io.ktor.client.features.*
+import io.ktor.client.features.json.*
+import io.ktor.client.request.*
+import io.ktor.client.statement.*
+import io.ktor.http.*
+import net.logstash.logback.argument.StructuredArguments
+import no.nav.syfo.domain.Fodselsnummer
+import no.nav.syfo.util.bearerHeader
+import org.slf4j.LoggerFactory
+
+class Tilgangskontroll(
+    private val url: String = "http://syfo-tilgangskontroll/syfo-tilgangskontroll/api/tilgang/bruker"
```

Comment:
Fint om du kan bytte til ingress-url i stedet for servicelocator-url, blir det mer sømløst ved evt. flytting til GCP i fremtiden.